main.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link href="style.css" rel="stylesheet">
+        <title>Виджет</title>
+    </head>
+    <body>
+        
+        <div class="mainblock">
+        	<div id="block1" class="blocks"><p><img src="images/weather/sun.png"></p><p>chicago / 23°</p></div>
```

Comment:
code formatting can be a little better, so each p and img etc should occupy a new line